src/UnitTest/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+cmake_minimum_required(VERSION 3.0)
+
+# Locate GTest
+find_package(GTest REQUIRED)
+include_directories(${GTEST_INCLUDE_DIRS})
+
+# Specify the unit tests
+add_executable(unitTests
+               main.cpp
+               Image_unittest.cpp)
```

Comment:
I think `image.cpp` would be fine, as it is under UnitTest folder.

Reply:
yeah, we could do that.